backend/src/routes/v0/tool.py

```
@@ -1,28 +1,45 @@
+
+This module provides FastAPI routes for managing and interacting with various
+tool services including built-in tools, Arcade, MCP, and A2A integrations.
+"""
+
+from typing import Literal
 from fastapi import status, Depends, APIRouter, Query
 from fastapi.responses import JSONResponse
 from sqlalchemy.ext.asyncio import AsyncSession
 
 from src.constants.description import Description
 from src.constants.examples import A2A_GET_AGENT_CARD_EXAMPLE, MCP_REQ_BODY_EXAMPLE, ARCADE_RESPONSE_EXAMPLE
```

Ruff (0.12.2)

`src.constants.examples.MCP_REQ_BODY_EXAMPLE` imported but unused

Remove unused import: `src.constants.examples.MCP_REQ_BODY_EXAMPLE`

(F401)